src/site/content/en/blog/local-fonts/index.md

```diff
@@ -254,7 +223,7 @@ TrueType, OpenType, Web Open Font Format (WOFF) fonts and others.
 
 ```js
 try {
-  const pickedFonts = await self.queryLocalFonts({
+  const pickedFonts = await window.queryLocalFonts({
     postscriptNames: ['Comic Sans MS'],
```

Comment:
Correct postscriptname seems to be `ComicSansMS`. Can you confirm on your side?

Reply:
Confirmed.